addons/mail/models/mail_template.py

```diff
@@ -121,6 +123,11 @@ def _compute_template_category(self):
                 else:
                     template.template_category = 'custom_template'
 
+    def _compute_current_user_is_template_editor(self):
```

Comment:
:socks: to define after '_compute_can_write', to keep compute methods ordered like fields